src/Braille/BrailleData.ts

```diff
@@ -1,88 +1,68 @@
+class BrailleData extends EncodingDataBase<BrailleEncoding> {
   public static readonly instance: BrailleData = new BrailleData();
-  private readonly _entries: BrailleEntry[] = [];
 
   constructor() {
-    this.addToList(BrailleEncoding.LetterA, BrailleCategory.Letter, 'A');
-    this.addToList(BrailleEncoding.LetterB, BrailleCategory.Letter, 'B');
-    this.addToList(BrailleEncoding.LetterC, BrailleCategory.Letter, 'C');
-    this.addToList(BrailleEncoding.LetterD, BrailleCategory.Letter, 'D');
-    this.addToList(BrailleEncoding.LetterE, BrailleCategory.Letter, 'E');
-    this.addToList(BrailleEncoding.LetterF, BrailleCategory.Letter, 'F');
-    this.addToList(BrailleEncoding.LetterG, BrailleCategory.Letter, 'G');
-    this.addToList(BrailleEncoding.LetterH, BrailleCategory.Letter, 'H');
-    this.addToList(BrailleEncoding.LetterI, BrailleCategory.Letter, 'I');
-    this.addToList(BrailleEncoding.LetterJ, BrailleCategory.Letter, 'J');
-    this.addToList(BrailleEncoding.LetterK, BrailleCategory.Letter, 'K');
-    this.addToList(BrailleEncoding.LetterL, BrailleCategory.Letter, 'L');
-    this.addToList(BrailleEncoding.LetterM, BrailleCategory.Letter, 'M');
-    this.addToList(BrailleEncoding.LetterN, BrailleCategory.Letter, 'N');
-    this.addToList(BrailleEncoding.LetterO, BrailleCategory.Letter, 'O');
-    this.addToList(BrailleEncoding.LetterP, BrailleCategory.Letter, 'P');
-    this.addToList(BrailleEncoding.LetterQ, BrailleCategory.Letter, 'Q');
-    this.addToList(BrailleEncoding.LetterR, BrailleCategory.Letter, 'R');
-    this.addToList(BrailleEncoding.LetterS, BrailleCategory.Letter, 'S');
-    this.addToList(BrailleEncoding.LetterT, BrailleCategory.Letter, 'T');
-    this.addToList(BrailleEncoding.LetterU, BrailleCategory.Letter, 'U');
-    this.addToList(BrailleEncoding.LetterV, BrailleCategory.Letter, 'V');
-    this.addToList(BrailleEncoding.LetterW, BrailleCategory.Letter, 'W');
-    this.addToList(BrailleEncoding.LetterX, BrailleCategory.Letter, 'X');
-    this.addToList(BrailleEncoding.LetterY, BrailleCategory.Letter, 'Y');
-    this.addToList(BrailleEncoding.LetterZ, BrailleCategory.Letter, 'Z');
+    super();
 
-    this.addToList(BrailleEncoding.Number0, BrailleCategory.Number, '0');
-    this.addToList(BrailleEncoding.Number1, BrailleCategory.Number, '1');
-    this.addToList(BrailleEncoding.Number2, BrailleCategory.Number, '2');
-    this.addToList(BrailleEncoding.Number3, BrailleCategory.Number, '3');
-    this.addToList(BrailleEncoding.Number4, BrailleCategory.Number, '4');
-    this.addToList(BrailleEncoding.Number5, BrailleCategory.Number, '5');
-    this.addToList(BrailleEncoding.Number6, BrailleCategory.Number, '6');
-    this.addToList(BrailleEncoding.Number7, BrailleCategory.Number, '7');
-    this.addToList(BrailleEncoding.Number8, BrailleCategory.Number, '8');
-    this.addToList(BrailleEncoding.Number9, BrailleCategory.Number, '9');
+    this.addToList(BrailleEncoding.LetterA, EncodingCategory.Letter, 'A');
```

Comment:
If only Javascript had Preprocessor Macros, then you could have changed 2 lines instead of 36 here.  ;)